src/modules/endpoint/models/EventModel.js

```diff
@@ -1,63 +1,93 @@
+// @flow
+
+import moment from 'moment'
+
+type eventModel = {
```

Comment:
This here looks a bit like duplicate code. So basically eventModel is a EventModel without methods? Looks redundant to me.

Reply:
already in work :D